docs/API/v3/service-index.md

```diff
@@ -0,0 +1,114 @@
+---
+# required metadata 
+
+title: Service Index, NuGet V3 API | Microsoft Docs
+author:
+- joelverhagen
+- kraigb
+ms.author:
+- joelverhagen
+- kraigb
+manager: skofman
+ms.date: 10/26/2017
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+ms.assetid: 2f6d6cf2-53fb-417a-b1d8-e0ac591c1699
+
+# optional metadata
+
+description: The V3 service index is the entry point of the NuGet V3 and enumerates the capabilities of the V3 server.
+keywords: NuGet V3 API entry point, NuGet V3 API endpoint discovery
+ms.reviewer:
+- karann
+- unnir
+
+---
+
+# Service Index
+
+The V3 service index is a JSON document that is the entry point for a V3 NuGet package source and allows a client
+implementation to discover the package source's capabilities. The service index is a JSON object with two required
+properties: `version` (the schema version of the service index) and `resources`  (the endpoints or capabilities of the
+package source).
+
+nuget.org's service index is located here:
+```
+https://api.nuget.org/v3/index.json
+```
+
+The service index for nuget.org looks something like is this:
+
+[!code-JSON [service-index.json](./_data/service-index.json)]
+
+## Versioning
+
+The `version` value is a SemVer 2.0.0 parseable version string which indicates the schema version of the service index.
+The V3 API mandates that the version string has a major version number of `3`. As non-breaking changes are made to the
+service index schema, the version string's minor version will be increased.
+
+Each resource in the service index is versioned independently from the service index schema version.
+
+The current schema version is `3.0.0-beta.1`.
```

Comment:
Even more confusion :) So, we have a NuGet v3 API, with protocol version 4.1.0 and schema version 3.0.0-beta.1. Think we need to do something about that :)